RecoMET/METProducers/python/CaloRecHitsBeamHaloCleaned_cfi.py

```diff
@@ -0,0 +1,9 @@
+import FWCore.ParameterSet.Config as cms
+
+CaloRecHitsBeamHaloCleaned = cms.EDProducer('CaloRecHitsBeamHaloCleaned',
```

Comment:
please use lower case for class module instance names 
and have the file name also start with the lower case letter